src/Microsoft.ApplicationInsights.AspNetCore/Extensions/ApplicationInsightsExtensions.cs

```diff
@@ -146,6 +146,8 @@ public static IServiceCollection AddApplicationInsightsTelemetry(this IServiceCo
                     excludedDomains.Add("core.chinacloudapi.cn");
                     excludedDomains.Add("core.cloudapi.de");
                     excludedDomains.Add("core.usgovcloudapi.net");
+                    excludedDomains.Add("localhost");
+                    excludedDomains.Add("127.0.0.1");
```

Comment:
Are we sure we want these always excluded?  It was mentioned in the issue that this will break tracking local dependencies when a developer sets up everything on a single machine for local development and testing.  I don't think this is something that should be universally forced on everyone.  I know a lot of developers used to run a "onebox" with all of their services and databases installed locally when testing a component or service that was part of a larger SOA system.  This configuration would exclude tracking such a development setup.